packages/components/src/styles/components/modal.scss

```diff
@@ -8,13 +8,18 @@
 //
 
 .hds-modal {
+  all: revert; // this must be first
```

Comment:
same comment as for the `Flyout`, maybe is better to just set the fixed position (even if I tried and it's not so obvious, I'll get back to this in a following comment)